mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/job/JobActor.java

```diff
@@ -1259,6 +1259,7 @@ class WorkerManager implements IWorkerManager {
                 .expireAfterWrite(1, TimeUnit.HOURS)
                 .build();
         private volatile boolean stageAssignmentPotentiallyChanged;
+        private final boolean batchSchedulingEnabled = ConfigurationProvider.getConfig().isBatchSchedulingEnabled();
```

Comment:
why not pass in the arg like other params?

Reply:
+1